source/languages.ttl

```diff
@@ -0,0 +1,3174 @@
+    skos:altLabel "Medelpersiska"@sv ;
+    skos:notation "pal",
+        "pal"^^kbv:ISO639-2,
+        "pal"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Pahlavi"@sv .
+
+<pam> a schema:Language ;
+    skos:notation "pam",
+        "pam"^^kbv:ISO639-2,
+        "pam"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Pampanga"@sv .
+
+<pan> a schema:Language ;
+    skos:altLabel "Panjabi"@sv ;
+    skos:notation "pan",
+        "pan"^^kbv:ISO639-2,
+        "pan"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Punjabi"@sv .
+
+<pap> a schema:Language ;
+    skos:notation "pap",
+        "pap"^^kbv:ISO639-2,
+        "pap"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Papiamento"@sv .
+
+<pau> a schema:Language ;
+    skos:notation "pau",
+        "pau"^^kbv:ISO639-2,
+        "pau"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Palauiska"@sv .
+
+<peo> a schema:Language ;
+    rdfs:comment "Ca 600-400 f.Kr."@sv ;
+    skos:notation "peo",
+        "peo"^^kbv:ISO639-2,
+        "peo"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Fornpersiska"@sv .
+
+<per> a schema:Language ;
+    skos:notation "per",
+        "per"^^kbv:ISO639-2,
+        "per"^^kbv:ISO639-2-B,
+        "fas"^^kbv:ISO639-2-T,
+        "fas"^^kbv:ISO639-3 ;
+    skos:prefLabel "Persiska"@sv .
+
+<phi> a schema:Language ;
+    skos:notation "phi",
+        "phi"^^kbv:ISO639-2,
+        "phi"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Filippinska språk (övriga)"@sv .
+
+<phn> a schema:Language ;
+    skos:altLabel "Puniska"@sv ;
+    skos:notation "phn",
+        "phn"^^kbv:ISO639-2,
+        "phn"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Feniciska"@sv .
+
+<pli> a schema:Language ;
+    skos:notation "pli",
+        "pli"^^kbv:ISO639-2,
+        "pli"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Pali"@sv .
+
+<pol> a schema:Language ;
+    skos:notation "pol",
+        "pol"^^kbv:ISO639-2,
+        "pol"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Polska"@sv .
+
+<pon> a schema:Language ;
+    skos:altLabel "Ponape"@sv ;
+    skos:notation "pon",
+        "pon"^^kbv:ISO639-2,
+        "pon"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Pohnpei"@sv .
+
+<por> a schema:Language ;
+    skos:notation "por",
+        "por"^^kbv:ISO639-2,
+        "por"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Portugisiska"@sv .
+
+<pra> a schema:Language ;
+    skos:notation "pra",
+        "pra"^^kbv:ISO639-2,
+        "pra"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Prakritspråk"@sv .
+
+<pro> a schema:Language ;
+    skos:altLabel "Provençalska"@sv ;
+    skos:notation "pro",
+        "pro"^^kbv:ISO639-2,
+        "pro"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Provensalska"@sv .
+
+<pus> a schema:Language ;
+    skos:notation "pus",
+        "pus"^^kbv:ISO639-2,
+        "pus"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Pashto"@sv .
+
+<que> a schema:Language ;
+    skos:notation "que",
+        "que"^^kbv:ISO639-2,
+        "que"^^kbv:ISO639-2-B ;
+    skos:prefLabel "Quechua"@sv .
+
+<raj> a schema:Language ;
+    owl:sameAs <rai> ;
+    skos:notation "rai",
+        "raj",
```

Comment:
```suggestion
    skos:notation "raj",
```

From https://github.com/libris/definitions/blob/develop/source/spraakkoder.tsv#L366

I can't find any reference to this code in 
https://www.loc.gov/standards/iso639-2/php/code_changes.php
https://www.loc.gov/marc/languages/language_code.html
https://www.loc.gov/marc/languages/languagechg.html
https://katalogverk.kb.se/katalogisering/Formathandboken/Sprakkoder/index.html

Is it an ancient MARC code? I suggest we just drop it.

Reply:
There is no usage in Libris so I suggest drop but keep the rai comment as a historyNote.